spec/app/models/mdm/web_page_spec.rb

```diff
@@ -3,6 +3,48 @@
 
   context 'associations' do
     it { is_expected.to belong_to(:web_site).class_name('Mdm::WebSite') }
+
+    context 'serialized cookie attribute' do
+      let(:valid_cookie) do
+        {
+          name: 'test name',
+          value: 'test value'
+        }
+      end
+
+      let(:invalid_cookie) do
```

Comment:
I'm worried that you fixed the problem that the import didn't work, but now if any code tries to use the cookie that will break so you'll have caused a bug farther down.  You should audit where WebPage cookies are used to see how the cookie is read, but I can't see code that expects a Hash just working.  Using a simple test case, you might get behavior that doesn't error, but is totally wrong since `Hash#[](String)` and `Hash#[](String)` work:

    h = {name: "foo"}
    s = "name: 'foo'"
    # symbol lookup
    h[:name] # 'foo'
    s[:name] # TypeError: no implicit conversion of Symbol into Integer
    # String lookup
    h['name'] # 'foo' (assuming HashWithIndifferentAccess)
    s['name'] # 'name'

See, the problem is that `String#[](String)` returns the piece of the string that matches the String or Regexp, so when you lookup a String in s you just get the String back.  That's going to be a much harder bug to track down.